.flowconfig

```diff
@@ -7,8 +7,8 @@
 .*/.nyc_output/.*
 
 [libs]
-./node_modules/flow-interfaces/interfaces/
 ./interfaces/
+./flow-typed/
```

Comment:
Can't we just move flow-typed defs in "interfaces"? I don't want a mess at the root of the project :)